client/components/tooltip/style.scss

```diff
@@ -1,3 +1,7 @@
+@import "@automattic/typography/styles/variables";
+
+$post-trends-box-shadow-color: #0000000d;
```

Comment:
Do you think that using rgba value here would be easier to understand?

Reply:
Will do!